tools/lint.py

```
@@ -2561,14 +2571,15 @@ def check_doctrine_callout(root: Path) -> list[str]:
         (RUNS_SCRIPT, "does not run tools/doctrine_callout.py"),
         (GATED_ON_PR, "is not gated on a pull_request event"),
         (FULL_HISTORY, "does not check out full history (`fetch-depth: 0`), "
-                       "so the base revision is unreadable and the callout "
-                       "loses this PR's own movement"),
+                       "so the base revision is unreadable: the callout loses "
+                       "this PR's movement AND the frontmatter arm raises, "
+                       "reddening this job on every pull request"),
```

Ruff (0.16.3)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)